src/helpers/xml-builder.js

```diff
@@ -1,2390 +1,2473 @@
+  let size = 0;
+  let stroke = 'single';
+  let color = '000000';
+
+  for (let tokenIdx = 0; tokenIdx < tokens.length; tokenIdx++) {
+    const token = tokens[tokenIdx];
+    // Accepted HTML Values for border style: https://developer.mozilla.org/en-US/docs/Web/CSS/border-style
+    if (
+      [
+        'solid',
+        'dashed',
+        'dotted',
+        'double',
+        'groove',
+        'ridges',
+        'inset',
+        'outset',
+        'hidden',
+        'none',
+        'windowtext', // tinyMCE has this border property
+      ].includes(token)
+    ) {
+      // Accepted OOXML Values for border style: http://officeopenxml.com/WPtableBorders.php
+      if (['dashed', 'dotted', 'double', 'inset', 'outset'].includes(token)) {
+        stroke = token;
+      } else if (['hidden', 'none'].includes(token)) {
+        stroke = ' nil';
+      } else {
+        stroke = 'single';
+      }
+    } else if (pointRegex.test(token)) {
+      const matchedParts = token.match(pointRegex);
+      // convert point to eighth of a point
+      size = pointToEIP(matchedParts[1]);
+    } else if (pixelRegex.test(token)) {
+      const matchedParts = token.match(pixelRegex);
+      // convert pixels to eighth of a point
+      size = pixelToEIP(matchedParts[1]);
+    } else {
+      color = fixupColorCode(token).toUpperCase();
+    }
+  }
+  // Syntax used for border color is either hsl or rgb
+  if (tokens.length !== 3) {
+    const openingBracketIdx = borderString.indexOf('(');
+    const closingBracketIdx = borderString.indexOf(')');
+    color = borderString.substring(openingBracketIdx - 3, closingBracketIdx + 1);
+    color = fixupColorCode(color).toUpperCase();
+  }
+  return [size, stroke, color];
+};
+
+const modifiedStyleAttributesBuilder = (docxDocumentInstance, vNode, attributes, options) => {
+  const modifiedAttributes = { ...attributes };
+
+  // styles
+  if (isVNode(vNode) && vNode.properties && vNode.properties.style) {
+    if (vNode.properties.style.color && !colorlessColors.includes(vNode.properties.style.color)) {
+      modifiedAttributes.color = fixupColorCode(vNode.properties.style.color);
+    }
+
+    if (
+      vNode.properties.style['background-color'] &&
+      !colorlessColors.includes(vNode.properties.style['background-color'])
+    ) {
+      modifiedAttributes.backgroundColor = fixupColorCode(
+        vNode.properties.style['background-color']
+      );
+    }
+
+    if (
+      vNode.properties.style.background &&
+      !colorlessColors.includes(vNode.properties.style.background)
+    ) {
+      modifiedAttributes.backgroundColor = fixupColorCode(vNode.properties.style.background);
+    }
+
+    if (
+      vNode.properties.style['vertical-align'] &&
+      verticalAlignValues.includes(vNode.properties.style['vertical-align'])
+    ) {
+      modifiedAttributes.verticalAlign = vNode.properties.style['vertical-align'];
+    }
+
+    if (
+      vNode.properties.attributes.valign &&
```

Comment:
@K-Kumar-01 Why change the vnode.properties.style['objectname'] syntax here? It's not consistent with the rest of the code. If this is something you feel strongly about, we can make this a git issue and do it consistently for everything, but keep in mind, fields like ['text-align'] need this syntax.

For now, undo this style change to make it more consistent